### PR TITLE
#8152: Update prelu sweep config

### PR DESCRIPTION
**Pull request by mcw-anasuya**
### Ticket
#8152 

### Problem description
Documentation incomplete for Prelu op with supported params

### What's changed
Updated supported params and moved sweep config

### Issue faced
Pybind is currently not available. Documentation will be updated when PreLU op is implemented correctly #8544 

### Checklist
- [x] [Post commit CI passes](https://github.com/tenstorrent/tt-metal/actions/runs/11235700656)
- [x] [Run Sweeps](https://github.com/tenstorrent/tt-metal/actions/runs/11186120034)

### CODE REVIEW

File: tests/sweep_framework/sweeps/eltwise/unary/prelu/prelu.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+# Parameters provided to the test vector generator are defined here.
+# They are defined as dict-type suites that contain the arguments to the run function as keys, and lists of possible inputs as values.
+# Each suite has a key name (in this case "suite_1") which will associate the test vectors to this specific suite of inputs.
+# Developers can create their own generator functions and pass them to the parameters as inputs.
+parameters = {
+    "nightly": {
+        "batch_sizes": [(1,)],
+        "height": [384, 1024],
+        "width": [1024, 4096],
```
Comment (Aswinmcw) [inline tests/sweep_framework/sweeps/eltwise/unary/prelu/prelu.py:27]: can we make it as a 4D tensor?

**Review by umadevimcw — APPROVED** [review]

File: tests/sweep_framework/sweeps/eltwise/unary/prelu/prelu.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+# Returns False, None if the vector is valid, and True, str with a reason for invalidation if it is invalid.
+def invalidate_vector(test_vector) -> Tuple[bool, Optional[str]]:
+    if test_vector["layout"] == ttnn.ROW_MAJOR_LAYOUT:
+        return True, "Row Major layout is not supported"
+    return False, None
+
+
+def torch_prelu(x, *args, **kwargs):
+    weight = kwargs.pop("scalar")
+    result = torch.nn.functional.prelu(x, torch.tensor(weight, dtype=x.dtype))
```
Comment (umadevimcw) [inline tests/sweep_framework/sweeps/eltwise/unary/prelu/prelu.py:48]: Is the golden function available for prelu?

Comment (mcw-anasuya) [reply tests/sweep_framework/sweeps/eltwise/unary/prelu/prelu.py:50]: No, it isn't.
I think we need to address that in #8544

**Review by VirdhatchaniKN — APPROVED** [review]

**Review by jdesousa-TT — APPROVED** [review]
Pls run sweeps pipeline before merging

**Review by eyonland — APPROVED** [review]

**Pull request merged**